### PR TITLE
[MX-306] fixed new works for you rail sorting

### PR DESCRIPTION
**Pull request by MounirDhahri**
The type of this PR is: **Bugfix**

This PR resolves [MX-306]

### Description

Fix new works for you rail sorting to match the home screen

Screenshot 2020-09-11 at 11 15 49
Screenshot 2020-09-11 at 11 15 57

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have added tests for my changes, or my changes don't require testing, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/master/docs/adding_state_migrations.md))

[MX-434]: https://artsyproduct.atlassian.net/browse/MX-434
[MX-306]: https://artsyproduct.atlassian.net/browse/MX-306

### CODE REVIEW

File: Podfile.lock [hunk 1]
```diff
@@ -865,6 +865,6 @@ SPEC CHECKSUMS:
   "XCTest+OHHTTPStubSuiteCleanUp": 4469ec8863c6bc022c5089a9b94233eb3416c5ee
   Yoga: 50fb6eb13d2152e7363293ff603385db380815b1
 
-PODFILE CHECKSUM: efaf566d47a3af74126258d469a41e3cae94e826
+PODFILE CHECKSUM: e27b74b5b054ccf36dfca674e5bea3f7d0383fcd
```
Comment (MounirDhahri) [inline Podfile.lock:868]: the Podfile got updated when I installed react-native-webview

File: src/lib/Containers/WorksForYou.tsx [hunk 2]
```diff
@@ -135,7 +135,7 @@ export const WorksForYouContainer = createPaginationContainer(
       @argumentDefinitions(
         count: { type: "Int", defaultValue: 10 }
         cursor: { type: "String" }
-        sort: { type: "ArtworkSorts", defaultValue: PARTNER_UPDATED_AT_DESC }
+        sort: { type: "ArtworkSorts", defaultValue: PUBLISHED_AT_DESC }
```
Comment (MounirDhahri) [inline src/lib/Containers/WorksForYou.tsx:138]: Now it's matching force, see https://github.com/artsy/force/blob/fe83a9b35ca9766df8c8f14491c3a5b700b80765/src/v2/Apps/WorksForYou/WorksForYouFeed.tsx#L137

Comment (ds300) [inline src/lib/Containers/WorksForYou.tsx:138]: 🙏

File: Podfile.lock (hunk 1 above)
Comment (ds300) [inline Podfile.lock:868]: hmm any ideas why this came up? Do you get this when you run `pod install` on master?

**Review by ds300 — APPROVED** [review]

File: Podfile.lock (hunk 1 above)
Comment (MounirDhahri) [reply Podfile.lock:868]: Yes, it got updated after installing `react-native-webview` which was merged yesterday
(Oh, I left a comment about this and forgot to submit it 🤦‍♂️)

**Pull request merged**